packages/middleware/src/lib/middleware.md

```diff
@@ -0,0 +1,14 @@
+# Middleware
```

Comment:
what is that file? Doesn't this belong into the design wiki?

Reply:
not sure, how this file ended up here?! @lukas-rotsching 

@MPSebastian Do you know more?